File: MultiplicationTables/MultiplicationTables.cpp
// Multiplication Tables program written in C++ for CodeEval.com by Steven A. Dunn

#include <cstdio>
#include <iostream>
#include <string>

using namespace std;

int main (int argc, int argv[])
{
	for(int i = 1; i <= 12; i++) 
	{
		for(int j = 1; j <= 12; j++) 
		{
			if(j == 1)
				printf("%2d", i*j);
			else
				printf("%4d", i*j);
		}
		cout << endl;
	}	
}

File: MultiplicationTables/c#/MultiplicationTables.cs
// multiplication tables soln in c# for code eval by steven a dunn

using System;

class MultiplicationTables {
	public static void Main(string[] args) {
		for (int i = 1; i < 13; i++) {
			for (int j = 1; j < 13; j++) {
				if (j == 1) 
					Console.Write("{0,2}", i * j);
				else
					Console.Write("{0,4}", i * j);
			}	
			Console.WriteLine();
		}
	}
}